README.md

```diff
@@ -33,7 +33,7 @@ let h_new = bcrypt::hash("password").unwrap();
 // Verify a password against an existing hash.
 let h = "$2y$05$bvIG6Nmid91Mu9RcmmWZfO\
          5HJIMCT8riNW0hEp8f6/FuA2/mHZFpe";
-assert_eq!(bcrypt::verify("password", h), true);
+assert!(bcrypt::verify("password", h));
```

Comment:
IMO `Eq`/`PartialEq` always come with some semantics attached (the whole point of having a typesystem!), and are not bound to be a simple `memcmp`.

The only reason why I didn't suggest implementing those traits is because checking the password comes with a computational cost that one wouldn't expect from `==`, hence the separate method.